.rubocop_todo.yml

```diff
@@ -40,6 +40,7 @@ Metrics/AbcSize:
   Max: 20
   Exclude:
     - 'lib/vagrant-service-manager/command.rb'
+    - 'plugins/guests/redhat/cap/box_version.rb'
```

Comment:
Add additional `if` increase the [Abc size](http://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize) by 0.60 (20.60/20).

Condition is now required to check whether data is empty or not  as from vagrant 1.8.4+ this block execute twice for Windows OS.

One solution could be to remove map which might have increase Abc size here.